recipe/meta.yaml

```diff
@@ -31,7 +31,7 @@ requirements:
     - python >=3.6
     - jupyter_core
     - jupyterlab_server >=2.0.0rc1,<3.0.0
```

Comment:
I put in a PR to update jupyterlab_server, and added you (@jtpio) as a maintainer. If you accept, would you comment on the PR? https://github.com/conda-forge/jupyterlab_server-feedstock/pull/32

Then we can bump this to match the python requirement

Reply:
Thanks, I commented there :+1: